x-pack/plugins/observability/public/pages/home/index.tsx

```diff
@@ -0,0 +1,259 @@
+      defaultMessage:
+        'Already using the Elastic Stack for logs? Add metrics in just a few steps and correlate metrics and logs in one place.',
+    }),
+  },
+  {
+    id: 'uptime',
+    title: i18n.translate('xpack.observability.section.apps.uptime.title', {
+      defaultMessage: 'Uptime',
+    }),
+    icon: 'logoUptime',
+    description: i18n.translate('xpack.observability.section.apps.uptime.description', {
+      defaultMessage:
+        'React to availability issues across your apps and services before they affect users.',
+    }),
+  },
+];
+
+const tryItOutItemsSection: ISection[] = [
+  {
+    id: 'demo',
+    title: i18n.translate('xpack.observability.section.tryItOut.demo.title', {
+      defaultMessage: 'Demo Playground',
+    }),
+    icon: 'play',
+    description: '',
+    href: 'https://demo.elastic.co/',
+    target: '_blank',
+  },
+  {
+    id: 'sampleData',
+    title: i18n.translate('xpack.observability.section.tryItOut.sampleData.title', {
+      defaultMessage: 'Add sample data',
+    }),
+    icon: 'documents',
+    description: '',
+    href: '/app/home#/tutorial_directory/sampleData',
+  },
+];
+
+const Container = styled.div`
+  min-height: calc(100vh - 48px);
+  background: ${(props) => props.theme.eui.euiColorEmptyShade};
+`;
+
+const Title = styled.div`
+  background-color: ${(props) => props.theme.eui.euiPageBackgroundColor};
+  border-bottom: ${(props) => props.theme.eui.euiBorderThin};
+`;
+
+const Page = styled.div`
+  width: 100%;
+  max-width: 1200px;
+  margin: 0 auto;
+  overflow-y: hidden;
+}
+`;
+
+export const Home = () => {
+  const { core } = usePluginContext();
+
+  useEffect(
+    () => {
+      core.chrome.setBreadcrumbs([
+        {
+          text: i18n.translate('xpack.observability.home.breadcrumb.observability', {
+            defaultMessage: 'Observability',
+          }),
+        },
+        {
+          text: i18n.translate('xpack.observability.home.breadcrumb.gettingStarted', {
+            defaultMessage: 'Getting started',
+          }),
+        },
+      ]);
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    []
+  );
+
+  return (
+    <Container>
+      <Title>
+        <Page>
+          <EuiSpacer size="xxl" />
+          <EuiFlexGroup>
+            <EuiFlexItem grow={false}>
+              <EuiIcon type="logoObservability" size="xxl" />
+            </EuiFlexItem>
+            <EuiFlexItem>
+              <EuiTitle size="m">
+                <h1>
+                  {i18n.translate('xpack.observability.home.title', {
+                    defaultMessage: 'Observability',
+                  })}
+                </h1>
+              </EuiTitle>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+          <EuiSpacer size="xxl" />
+        </Page>
+      </Title>
+      <Page>
+        <EuiSpacer size="xxl" />
+        <EuiFlexGroup direction="column">
+          {/* title and description */}
+          <EuiFlexItem style={{ maxWidth: 567 }}>
```

Comment:
Unit - and the size seems oddly specific. Does it have to align with something?

Reply:
Removed to be 50% instead of fixed pixels.